cps/CP0/CP0.md

```diff
@@ -1 +1,284 @@
+
+**Cooling-off period**
+
+A cooling-off period is the time that must pass before a proposal can be re-submitted. 
+If a proposal fails to pass a snapshot vote, the cooling-off period is 15 days (counting from when the snapshot vote was created) before a new RFC can be resubmitted.
+
+The cooling-off period is important in order to take the feedback from the Community into account and make the necessary changes to the submitted proposal, and re-submit the RFC.
+
+
+### **Proposal types**
+We have defined the following proposal types (CPs) and assigned them each with a number.
+
+|CP #|Proposal type|Short description|
+| --- | --- | --- |
+|CP-1|Request for Mandate with Funding|Seeking mandate as a group/individual within the Centrifuge DAO to enact a project/work stream|
+|CP-1.1|Request for Mandate without Funding|Same as CP-1, just without initial funding|
+|CP-1.2|Removal of Mandate|Removal of a group’s mandate to enact a project/work stream (CP-1)|
+|CP-2|Request for Funding|Asking for funding from the Treasury|
+|CP-3|Runtime Upgrades|Proposals for Runtime Upgrades|
+|CP-3.1|Emergency Proposals|Emergency proposals in case of hacks, exploits, attacks, or network halt|
+|CP-4|General Improvements|Any proposal type, that does not fit under any of the other CPs|
+|CP-5|Centrifuge Pool Onboarding Proposal (POP)|Onboarding new pools on Centrifuge Chain|
+
+In general, the Governance process can be divided into two parts; *off-chain* and *on-chain Governance*.
+
+*Off-chain Governance:*
+This refers to the part of the process that takes place on our Forum (Request For Comments) and OpenSquare (snapshot voting). The purpose of this is generally to get input from the Community on a proposal and gather support, before moving the proposal on-chain (if necessary - not all proposals need to be submitted on-chain in order to pass).
+
+*On-chain Governance:*
+This refers to the part of the process that takes places on the blockchain. First a proposal is created on-chain (can be done by either the public or the Council) and this is generally followed by an on-chain referendum where all token holders can vote Aye or Nay.
+
+Some of these proposal types have different governance processes (see individual description of their processes below), but what they all  have in common is that they start with a post on the Forum and all CPs must be submitted to the [Centrifuge Proposals Repository](https://github.com/centrifuge/cps) on Github.
+
+It is very important that the CP is submitted correctly. In the event of an incorrect format of a proposal, the process will need to be repeated.
+
+Please contact the Governance & Coordination Group on the Forum or Discord ([Rhano](https://gov.centrifuge.io/u/Rhano) or [ImdioR](https://gov.centrifuge.io/u/imdior)), if you have any doubts or questions, and they will assist you.
+
+#### **CP-1 (MRF) Mandate Request with Funding**
```

Comment:
No need to mark them as bold:
```suggestion
#### CP-1 (MRF) Mandate Request with Funding
```

Reply:
Related, do you think you would add these CP-1-5 descriptions to the respective PRs as separate file?

Reply:
Do you mean instead of adding the descriptions here, or in addition to having them here?

Reply:
Removed bold from all proposal types